vector/src/main/java/im/vector/app/features/spaces/SpaceBetaHeaderItem.kt

```diff
@@ -16,27 +16,12 @@
 
 package im.vector.app.features.spaces
 
-import android.view.View
-import com.airbnb.epoxy.EpoxyAttribute
 import com.airbnb.epoxy.EpoxyModelClass
 import im.vector.app.R
-import im.vector.app.core.epoxy.ClickListener
 import im.vector.app.core.epoxy.VectorEpoxyHolder
 import im.vector.app.core.epoxy.VectorEpoxyModel
-import im.vector.app.core.epoxy.onClick
 
 @EpoxyModelClass(layout = R.layout.item_space_beta_header)
 abstract class SpaceBetaHeaderItem : VectorEpoxyModel<SpaceBetaHeaderItem.Holder>() {
-
-    @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash)
-    var clickAction: ClickListener? = null
-
-    override fun bind(holder: Holder) {
-        super.bind(holder)
-        holder.feedBackAction.onClick(clickAction)
-    }
-
-    class Holder : VectorEpoxyHolder() {
-        val feedBackAction by bind<View>(R.id.spaceBetaFeedbackAction)
-    }
+    class Holder : VectorEpoxyHolder()
```

Comment:
What do you think about completely deleting `SpaceBetaHeaderItem` class and `item_space_beta_header.xml` and improve `GenericHeaderItem` with a description?

Reply:
text appearance of title in `item_generic_header` is a little bit different, so it will require confirmation from designers. I'd say we should do that as a part of design system implementation, which should happen soon 🤔